css/ghost_host.css

```
@@ -0,0 +1,462 @@
+body {
+  margin: 0 auto;
+  width: 1600px;
+}
+
+header {
+  height: 1065px;
+  width: 1600px;
+  background-image: url(../img/header.png);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.wrapper {
+  margin: 0 auto;
+}
+
+.flexRow {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.headerNav {
+  padding-top: 25px;
+  width: 1200px;
+}
+
+.headerNavigation {
+  width: 645px;
+}
+
+.headerNavLink {
+  font-size: 16pt;
+  font-family: 'Lato';
```

```suggestion
  font-family: 'Lato', Helvetica, sans-serif;
```
Do like this to prevent mistakes in landing look, when file with font can't be loaded